commit 43845fa70fc751736c44c18f4cee42d49bfd1392
Author: Jia Tan <jiat0218@gmail.com>
Date:   Tue Jul 18 22:52:25 2023 +0800

    liblzma: Reword lzma_str_list_filters() documentation.
    
    This further improves the documentation from commit
    f36ca7982f6bd5e9827219ed4f3c5a1fbf5d7bdf. The previous wording of
    "supported options" was slightly misleading since the options that are
    printed are the ones that are relevant for encoding/decoding. It is not
    about which options can or must be specified.

diff --git a/src/liblzma/api/lzma/filter.h b/src/liblzma/api/lzma/filter.h
index 98b46df8..b38bc6ac 100644
--- a/src/liblzma/api/lzma/filter.h
+++ b/src/liblzma/api/lzma/filter.h
@@ -731,7 +731,7 @@ extern LZMA_API(lzma_ret) lzma_str_from_filters(
  *
  * If a filter_id is given then only one line is created which contains the
  * filter name. If LZMA_STR_ENCODER or LZMA_STR_DECODER is used then the
- * supported options for encoding or decoding are listed on the same line too.
+ * options read by the encoder or decoder are printed on the same line.
  *
  * If filter_id is LZMA_VLI_UNKNOWN then all supported .xz-compatible filters
  * are listed: